src/funding/funding.js

```diff
@@ -100,7 +100,7 @@ export function determineEligibleFunding({ fundingSource, layout, platform, fund
         return [ fundingSource ];
     }
 
-    let eligibleFunding = values(FUNDING).filter(source =>
+    let eligibleFunding = SUPPORTED_FUNDING_SOURCES.filter(source =>
```

Comment:
This looks good to me. `SUPPORTED_FUNDING_SOURCES` is already an array so we should use it and respect that order:
- https://github.com/paypal/paypal-funding-components/blob/c6df70ecf6c97cf31ececd254772974b747a658f/src/config.js#L7
- https://github.com/paypal/paypal-sdk-constants/blob/master/src/funding.js#L3